proto/node.proto

```diff
@@ -23,6 +23,7 @@ service Node {
   
   // TODO: add Acknowledgment route
   // for the client to say he successfully stored the quote_id
+  rpc Ack (AckRequest) returns (AckResponse);
```

Comment:
Can we use the full world `Acknowledge` instead?
I prefer not to use abbreviations in types and function name.